spec/ffi/long_double_spec.rb

```diff
@@ -6,13 +6,14 @@
 require File.expand_path(File.join(File.dirname(__FILE__), "spec_helper"))
 require 'bigdecimal'
 
-describe ":long_double arguments and return values" do
+# long double not yet supported on TruffleRuby
+describe ":long_double arguments and return values", :if => RUBY_ENGINE != "truffleruby" do
   module LibTest
     extend FFI::Library
     ffi_lib TestLibrary::PATH
     attach_function :add_f128, [ :long_double, :long_double ], :long_double
     attach_function :ret_f128, [ :long_double ], :long_double
-  end
+  end if RUBY_ENGINE != "truffleruby"
```

Comment:
Will this even run given the `:if` clause above?

Reply:
Yes, unfortunately this is needed (I though too it wouldn't be needed initially), and apparently RSpec's `if:` just skips the `it` examples but still yields to the block.

Reply:
👍weird, seems like the wrong choice but 🤷‍♂️